modules/proc_upgrade_standalone.adoc

```diff
@@ -64,15 +64,87 @@ Images for Quay 3.3.4 and earlier are available from link:https://quay.io[quay.i
 
 == Upgrade to 3.9.z from 3.8.z
 
-If you are upgrading your standalone {productname} deployment from 3.8.z -> 3.9, it is highly recommended that you upgrade PostgreSQL from version 10 -> 13. To upgrade PostgreSQL from 10 -> 13, you must bring down your PostgreSQL 10 database and run a migration script to initiate the process:
+If you are upgrading your standalone {productname} deployment from 3.8.z -> 3.9, it is highly recommended that you upgrade PostgreSQL from version 10 -> 13. To upgrade PostgreSQL from 10 -> 13, you must bring down your PostgreSQL 10 database and run a migration script to initiate the process. 
+
+Use the following procedure to upgrade PostgreSQL from 10 -> 13 on a standalone {productname} deployment. 
+
+.Procedure 
+
+. Enter the following command to scale down the {productname} container:
++
 [source,terminal]
 ----
-$ podman run -d --name postgresql_database \
+$ sudo podman stop <quay_container_name>
+----
+
+. Optional. If you are using Clair, enter the following command to stop the Clair container:
++
+[source,terminal]
+----
+$ sudo podman stop <clair_container_id>
+----
+
+. Run the Podman process from SCLOrg's link:https://github.com/sclorg/postgresql-container/tree/master/13#data-migration[Data Migration] procedure, which allows for data migration from a remote PostgreSQL server:
++
+[source,terminal]
+----
+$ sudo podman run -d --name <migration_postgresql_database> <1>
     -e POSTGRESQL_MIGRATION_REMOTE_HOST=172.17.0.2 \
     -e POSTGRESQL_MIGRATION_ADMIN_PASSWORD=remoteAdminP@ssword \
+    -v </host/data/directory:/var/lib/postgresql/data> <2>
```

Comment:
-v /host/data/directory:/var/lib/pgsql/data:Z

Reply:
A little bit more change here, to be more clear, we logged jira ticket to track this: https://issues.redhat.com/browse/PROJQUAY-5942